tools/buildsys/src/builder.rs

```diff
@@ -108,6 +109,12 @@ impl PackageBuilder {
             arch = arch,
         );
 
+        if let Some(image_features) = image_features {
+            for image_feature in image_features.iter() {
+                args.build_arg(format!("{}", image_feature), "1");
```

Comment:
Elsewhere I have comment on two things:
- Should the allowable image features be constrained?
- Should we allow for future image features that are not flags, i.e. where they have a string or numeric value.

Even if you keep image features as a list of unconstrained booleans, I think you need to double check that the value given is "true" since someone could write `grub-set-private-var = false` in their toml.

Reply:
The `image_features()` helper method has a filter where only "true" values are returned.

I briefly considered feature flags that aren't booleans, and figured that we could cross that bridge down the road when the use case was better understood.